lib/urlwatch/reporters.py

```diff
@@ -600,11 +600,29 @@ def submit(self):
 
         return result
 
+    @staticmethod
+    def _format_body(text: str) -> str:
+        return "```diff\n{}\n```".format(
+            text.translate(str.maketrans({'`': r'\`', '\\': r'\\'})))
```

Comment:
Not sure if this is the best way to escape this (see [Markdown syntax](https://daringfireball.net/projects/markdown/syntax)). If you use ` ``` ` as the delimiter, you might just need to escape 3 literal backticks in one line, not sure...

Reply:
All `` ``` `` inside escapes just fine. First the translation is done, then wrapped `` ```diff\n{}\n``` `` . The [Telegram Bot API][1] only wants backticks and backslashes escaped inside when using MarkdownV2 parse_mode. I have found an example over at [python-telegram-bot/python-telegram-bot][2], but instead of running the input through the regexp library I used `str.maketrans` / `str. translate`.

Telegram Desktop
(excuse the large screenshots)

![Screenshot_20210606_154655](https://user-images.githubusercontent.com/6209465/120927170-350dc180-c6e0-11eb-8fe3-0ed296b09fc4.png)


[1]: https://core.telegram.org/bots/api#markdownv2-style
[2]: https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/utils/helpers.py#L149